zerver/views/webhooks/pingdom.py

```diff
@@ -8,8 +8,8 @@
 import ujson
 
 
-PINGDOM_SUBJECT_TEMPLATE = 'Your Pingdom {name} descries something important.'
-PINGDOM_MESSAGE_TEMPLATE = 'Service {service_url} changed it\'s status from {previous_state} to {current_state}.'
+PINGDOM_SUBJECT_TEMPLATE = '{name}: {type} - from {previous_state} to {current_state}.'
```

Comment:
I think the subject should be just `{name} status`.  The reason is that it's actually a lot more useful for this sort of alerting integration if the UP=>DOWN change gets threaded with the same subject as the DOWN=>UP change.